.github/workflows/release.yml

```
@@ -0,0 +1,76 @@
+on:
+  push:
+    tags:
+      - 'release-*'
+
+name: build and release stable version
+
+jobs:
+  build-linux:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Setup rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly
+          override: true
+      - name: Release build
+        uses: actions-rs/cargo@v1
+        with:
+          command: build
+          args: --release
+      - name: Zip binary for Linux(x86_64)
+        run: |
+          zip cece-linux-x86_64 target/release/cece_rust
```

最新の master で変更になってます
```suggestion
          zip cece-linux-x86_64 target/release/cece
```